packages/desktop/package.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "desktop",
-    "productName": "Firefly",
+    "name": "firefly-desktop",
+    "productName": "Firefly Desktop",
```

Comment:
I believe this affects the naming of the app on desktop as well (therefore potentially impacting file paths), so we should do this change in a separate PR too. Testing this change will involve testing the entire update process to ensure app data is not lost when we change this, as well as ensuring CI and deployment scripts don't break